protobuf/v1/nexus.proto

```diff
@@ -325,13 +329,18 @@ message RebuildHistoryResponse {
 }
 
 message RebuildHistoryRecord {
-  string dst_uri = 1;                       // URI of the rebuilt replica (destination)
   string src_uri = 2;                       // URI of the source replica
-  RebuildJobState state = 3;                // Final state of the rebuild job
-  uint64 rebuilt_data_size = 4;             // Size of the data rebuilt
-  bool is_partial = 5;                      // Full or Partial rebuild
-  google.protobuf.Timestamp started_at = 6; // Start timestamp(UTC) of the rebuild job
-  google.protobuf.Timestamp ended_at = 7;   // End timestamp(UTC) of the rebuild job
+  string dst_uri = 1;                       // URI of the rebuilt replica (destination)
+  RebuildJobState state = 3;                // final state of the rebuild job
+  uint64 blocks_total = 8;                  // total number of blocks was to recover
```

Comment:
```suggestion
  uint64 blocks_total = 8;                  // total number of blocks to recover
```

Reply:
It's a history record, so it's past tense: "was to recover". It may be useful for cancelled/failed jobs.

Reply:
That doesn't sound correct wording for me, I'll let @blaisedias weigh in since he's our grammar expert :D

Reply:
'total number of blocks that were meant to be recovered'  :)  a little longer though

Reply:
okay, I will do as Tiago suggested "total number of blocks to recover". Same as on rebuild stats.

Reply:
I changed to "total number of blocks of the replicas"